local_sales/app/queries/users_query.rb

```diff
@@ -0,0 +1,5 @@
+class UsersQuery
+  def self.administrators
+    User.all.where(role: 'admin')
```

Comment:
Here you don't need to use the `all`.
```
User.where(role: 'admin')
```

And also, you can use model (active_record) scopes in this cases you only need to add this in your model:
```
  scope :admin, -> { where(role: 'admin') }
```
and to call this scope, you can call it on the class like this:
```
User.admin
```

You can check more info about the `scopes` on this link:
https://guides.rubyonrails.org/active_record_querying.html#scopes

Please let me know if you have any questions